src/wwwroot/app/views/settings/roles/roles.scss

```diff
@@ -66,4 +66,7 @@ ul.roles-tab li {
 
 .role-body {
    width: 102%;
```

Comment:
do you sure, that once 2% over will be enough for resolving problem? May be find other way for problem resolving?